### PR TITLE
fix #282724: add European Concert Band template

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
note to self: need to adjust to #4618

File: share/templates/07-Band_and_Percussion/09-European_Concert_Band.mscx [hunk 1]
```diff
@@ -0,0 +1,10855 @@
...
+          </Articulation>
+        <Articulation name="tenuto">
+          <velocity>100</velocity>
+          <gateTime>100</gateTime>
+          </Articulation>
+        <Articulation name="marcato">
+          <velocity>120</velocity>
+          <gateTime>67</gateTime>
+          </Articulation>
+        <Articulation name="sforzato">
```
Comment (anatoly-os) [inline share/templates/07-Band_and_Percussion/09-European_Concert_Band.mscx]: Could you please add new articulations in favour of #4618 ?

**Comment by Jojo-Schmitz** [comment]
I did, yesterday

File: share/templates/07-Band_and_Percussion/09-European_Concert_Band.mscx (hunk 1 above)
Comment (Jojo-Schmitz) [reply share/templates/07-Band_and_Percussion/09-European_Concert_Band.mscx]: I did, yesterday, so theae are in already

Comment (Jojo-Schmitz) [reply share/templates/07-Band_and_Percussion/09-European_Concert_Band.mscx]: Ah, I see, for this instrument too, not just the global settings.
OK, to follow soon

Comment (Jojo-Schmitz) [reply share/templates/07-Band_and_Percussion/09-European_Concert_Band.mscx]: done

**Comment by Jojo-Schmitz** [comment]
note to self: need to ajdust to #4625, i.e. add that template to instrumentxml.h

**Pull request merged**